packages/website/docs/adding-suggested-searches.md

```diff
@@ -0,0 +1,133 @@
+
+For now, `plugins` is an empty array, but you'll learn how to add the Query Suggestions plugin next.
+
+## Adding Query Suggestions
+
+The [`autocomplete-plugin-query-suggestions`](createQuerySuggestionsPlugin) package provides the [`createQuerySuggestionsPlugin`](createQuerySuggestionsPlugin) function for creating a Query Suggestions plugin out-of-the-box.
+
+It requires an [Algolia search client](https://www.algolia.com/doc/api-client/getting-started/install/javascript/) initialized with an [Algolia application ID and API key](https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/how-to/importing-with-the-api/#application-id) and an `indexName`. The `indexName` is the name of your [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) index. **You must have a populated [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/#implementing-query-suggestions) index to use this plugin.**
+
+```js
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+
+const searchClient = algoliasearch(
+  'YOUR_APP_ID',
+  'YOUR_SEARCH_API_KEY'
+);
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'YOUR_QUERY_SUGGESTIONS_INDEX_NAME',
+  },
+});
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+
+You can optionally pass a `getSearchParams` function to apply [Algolia query parameters](https://www.algolia.com/doc/api-reference/api-parameters/) to the suggestions returned from the plugin. For example, you may choose to display ten Query Suggestions if the user hasn't typed anything yet, but only five if they have. You may want to do this if you are [displaying other categories](creating-multi-source-autocompletes) along with Query Suggestions.
+
+```js
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+
+const searchClient = algoliasearch(
+  'YOUR_APP_ID',
+  'YOUR_SEARCH_API_KEY'
+);
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'YOUR_QUERY_SUGGESTIONS_INDEX_NAME',
+  getSearchParams({ state }) {
+    return { hitsPerPage: state.query ? 5 : 10 };
+  },
+});
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
```

Comment:
Should we add a note below with our credentials so that they can use this code snippet?